logstash-core/lib/logstash/modules/kibana_config.rb

```diff
@@ -19,6 +19,7 @@ def initialize(modul, settings)
     @name = modul.module_name
     @settings = settings
     @index_name = settings.fetch("dashboards.kibana_index", ".kibana")
+    @metrics_max_buckets = settings.fetch("dashboards.metrics_max_buckets", 60000)
```

Comment:
sounds good. can you please add a comment explaining the magic number? Also, please make it a constant.